app/models/console.rb

```diff
@@ -0,0 +1,16 @@
+class Console < ApplicationRecord
```

Comment:
Not sure I like this class name as it can be confused with the Rails::Console.  Not sure what a better name might be, though...RemoteConsole?